docs/src/reference/scripting.md

```diff
@@ -81,6 +81,36 @@ It explains #name.
 Sum is #add(2, 3).
 ```
 
+Let bindings can be used to destructure arrays and dictionaries.
+
+```example
+#let (x, y) = (1, 2)
+The coordinates are #x, #y.
+
+#let (_, c1, .., c2) = "Hello World!".clusters()
+The second cluster of "Hello World!" is #c1.
+The last cluster is #c2.
+
+#let (Homer, Austen: jane-austen, ..books) = (
+  "Shakespeare": "Hamlet",
+  "Homer": "The Odyssey",
+  "Dicken": "Oliver Twist",
+  "Austen": "Persuasion",
+)
+#for (author, title) in books [
+  #author wrote #title, 
+]
+Homer wrote #Homer and Austen wrote #jane-austen.
+```
```

Comment:
maybe split this up into two smaller exmaples, otherwise it's gonna be a bit overwhelming. also ideally, the lines would be a bit shorter so that the docs can be side-by-side, but that's hard to tell without the ability to build the docs. I can also do it myself at the end.

Reply:
I updated the docs, but the examples are still a bit clunky. I'll try to think of something better.